.github/workflows/native-it-selected-graalvm.yml

```diff
@@ -0,0 +1,396 @@
+name: Quarkus CI - Native IT on selected GraalVM
+
+on:
+  workflow_dispatch:
+    inputs:
+      BRANCH:
+        description: 'Branch to use'
+        required: true
+        default: 'main'
+        type: string
+      NATIVE_COMPILER:
+        description: 'the native compiler to use'
+        required: true
+        default: 'mandrel'
+        type: choice
+        options:
+          - mandrel
+          - graalvm-community
+          - graalvm
+          - liberica
+      NATIVE_COMPILER_VERSION:
+        description: 'the native compiler version to use'
+        required: true
+        default: '21'
+        type: choice
+        options:
+          - '17'
+          - '21'
+          - '22'
+
+env:
+  # Workaround testsuite locale issue
+  LANG: en_US.UTF-8
+  COMMON_MAVEN_ARGS: "-e -B --settings .github/mvn-settings.xml --fail-at-end"
+  COMMON_TEST_MAVEN_ARGS: "-Dformat.skip -Denforcer.skip -DskipDocs -Dforbiddenapis.skip -DskipExtensionValidation -DskipCodestartValidation"
+  NATIVE_TEST_MAVEN_ARGS: "-Dtest-containers -Dstart-containers -Dquarkus.native.native-image-xmx=6g -Dnative -Dnative.surefire.skip -Dno-descriptor-tests clean install -DskipDocs"
+  JVM_TEST_MAVEN_ARGS: "-Dtest-containers -Dstart-containers -Dquarkus.test.hang-detection-timeout=60"
+  PTS_MAVEN_ARGS: "-Ddevelocity.pts.enabled=${{ github.event_name == 'pull_request' && github.base_ref == 'main' && 'true' || 'false' }}"
+  DB_USER: hibernate_orm_test
+  DB_PASSWORD: hibernate_orm_test
+  DB_NAME: hibernate_orm_test
+  DEVELOCITY_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
```

Comment:
Since this is mostly meant to be used in forks I wonder whether the use of develocity could cause issues... Does it work fine even if we don't have an access key?

Reply:
Forks won't have the secret set up so I think we should be fine.